doc/devel/dependencies.rst

```diff
@@ -186,6 +186,12 @@ Required:
 - Ghostscript_ (>= 9.0, to render PDF files)
 - Inkscape_ (to render SVG files)
 
+.. note::
```

Comment:
Indent so that this is within the Inkscape bullet point.

Reply:
Fixed and seems to work!